doctrine/doctrine-bundle/2.13/manifest.json

```
@@ -26,7 +26,7 @@
                 "  environment:",
                 "    POSTGRES_DB: ${POSTGRES_DB:-app}",
                 "    # You should definitely change the password in production",
-                "    POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-!ChangeMe!}",
+                "    POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-\"!ChangeMe!\"}",
```

I suppose that the problem comes from the `!` char? Weirdly, we never had this problem before.

Anyway, what about making it simpler like (cc @dunglas):

```suggestion
                "    POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-ChangeMe}",
```